quadruped_pympc/controllers/Controllers.py

```diff
@@ -0,0 +1,543 @@
+import os
+import time
+
+# TODO: Ugly hack so people dont have to run the python command specifying the working directory.
```

Comment:
I think this hack is not needed anymore

Reply:
Remove this  and other unused dependencies from the code